corehq/apps/domain/models.py

```diff
@@ -681,7 +681,7 @@ def get_all(cls, include_docs=True):
 
     @classmethod
     def get_all_names(cls):
-        return [d['key'] for d in cls.get_all(include_docs=False)]
+        return sorted({d['key'] for d in cls.get_all(include_docs=False)})
```

Comment:
why `sorted()` and not `list()`?

Reply:
Using ```list()``` will result in an unsorted list, when the current behavior is to be sorted.